applications/LinearSolversApplication/custom_solvers/eigen_pardiso_lu_solver.h

```diff
@@ -43,11 +42,35 @@ class EigenPardisoLUSolver
 
     void Initialize(Parameters settings)
     {
+        // Set default parameters
+        if (!settings.Has("num_threads_mkl")) {
+            settings.AddEmptyValue("num_threads_mkl").SetString("do_nothing"); // Default to 0 (do nothing)
+        }
+
+        // Configure number of threads for MKL Pardiso solver
+        int number_of_mkl_threads = 0;
+        if (settings["num_threads_mkl"].IsNumber()) {
+            number_of_mkl_threads = settings["num_threads_mkl"].GetInt();
+        } else if (settings["num_threads_mkl"].GetString() == "minimal") {
+            number_of_mkl_threads = static_cast<int>(MKLUtilities::MKLThreadSetting::Minimal);
+        } else if (settings["num_threads_mkl"].GetString() == "consistent") {
+            number_of_mkl_threads = static_cast<int>(MKLUtilities::MKLThreadSetting::Consistent);
+        } else if (settings["num_threads_mkl"].GetString() == "do_nothing") {
+            number_of_mkl_threads = static_cast<int>(MKLUtilities::MKLThreadSetting::Do_nothing);
```

Comment:
shouldn't this be the ParallelUtils::GetNumThreads() to be consistent with openmp?